ntbs-service-unit-tests/DataMigration/NotificationMapperTest.cs

```diff
@@ -690,6 +690,22 @@ public async Task whenLastTreatmentEventIsOutcomeStillOnTreatment_DoNotCloseNoti
             // Assert
             Assert.Equal(NotificationStatus.Notified, notification.NotificationStatus);
         }
+        
+        [Fact]
+        public async Task correctlyMaps_tbServiceReferralReceievedDate_whenNotNull()
+        {
+            // Arrange
+            const string legacyId = "102-1";
```

Comment:
It doesn't really matter, but it would have been a little more robust to use a new notification for this test, rather than an existing one. When multiple tests use the same notification there's a risk that someone editing one test will accidentally make the other test fail. In practice, I really can't see that happening here though.

Reply:
I'll bear that in mind for the future!